### ISSUE TITLE
Add ability to filter files using a pattern

### ISSUE THREAD

**Issue by shaddyz** [lmtm/node-sync-files#3]
This tool is almost exactly what I need, however, it's lacking the ability to filter the files that are synchronized. I'd like to be able to direct the tool to only sync html files for instance.

**Comment by ghost** [comment]
Exactly! I like to sync folder but have to exclude some auto. generated files.

I suggest a syntax like this: `sync-files dir1/ dir2/ --exclude=*.css`

**Comment by gregjacobs** [comment]
This would be a great feature.

How about using the [minimatch](https://github.com/isaacs/minimatch) globbing syntax like other file utilities? Would be great to be able to write something like this:

```
sync-files "src/**/*.+(html|js)" "build/"
```

[node-glob](https://github.com/isaacs/node-glob) should be able to help with this

**Comment by AndyOGo** [comment]
@shaddyz  @dotmagic @gregjacobs 
I recently had the exact same issue.
So I developed my own glob based version.
go check it, hope it helps [`sync-glob`](https://github.com/AndyOGo/node-sync-glob)

**Comment by shaddyz** [comment]
@AndyOGo looks cool! unfortunately i'm working on a different project now so I don't have a need for it at the moment, but it looks like it will solve the problem. thank you! :+1:

**Comment by AndyOGo** [comment]
@shaddyz thanks, np. Yeah, I hope it solves it well^^
